integrations/amazon_bedrock/src/haystack_integrations/components/rankers/amazon_bedrock/ranker.py

```diff
@@ -14,6 +13,7 @@
 logger = logging.getLogger(__name__)
 
 MAX_NUM_DOCS_FOR_BEDROCK_RANKER = 1000
+SUPPORTED_REGIONS = ["us-west-2", "ap-northeast-1", "ca-central-1", "eu-central-1"]
```

Comment:
I'd also be in favor of removing this. Otherwise we run the risk of always being out of date whenever Bedrock adds more regions for support.

Reply:
For the same reason I thought raising an error is a bit strict. What do you think about a warning instead? Not needed?

Reply:
I also think not needed since the warning message would also quickly get out of date which would probably lead to confusion. 

And I believe that AWS usually provides decent error messages if you ask for a model in an unsupported region.